php/WC_Gateway_Blockonomics.php

```diff
@@ -241,7 +241,11 @@ public function check_blockonomics_callback()
                     update_post_meta($wc_order->get_id(), 'expected_btc_amount', $order['satoshi']/1.0e8);
                 }
                 update_option('blockonomics_orders', $orders);
+            }else{
+                exit("Error: order not found");
             }
+        }else{
+            exit();
```

Comment:
else and exit() here is uncessary as the function is anyway existing

Reply:
Removed in 3a92048